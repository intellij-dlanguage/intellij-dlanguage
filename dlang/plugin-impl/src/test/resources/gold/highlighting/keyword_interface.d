interface D
{
    void bar() { }  // error, implementation not allowed
    static void foo() { } // ok
    final void abc() { } // ok
}
