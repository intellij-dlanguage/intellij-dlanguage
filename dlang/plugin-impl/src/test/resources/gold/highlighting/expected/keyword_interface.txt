DlangTokenType.interface ('interface')
WHITE_SPACE (' ')
DlangTokenType.ID ('D')
WHITE_SPACE ('\n')
DlangTokenType.{ ('{')
WHITE_SPACE ('\n')
WHITE_SPACE ('    ')
DlangTokenType.void ('void')
WHITE_SPACE (' ')
DlangTokenType.ID ('bar')
DlangTokenType.( ('(')
DlangTokenType.) (')')
WHITE_SPACE (' ')
DlangTokenType.{ ('{')
WHITE_SPACE (' ')
DlangTokenType.} ('}')
WHITE_SPACE ('  ')
DlangTokenType.LINE_COMMENT ('// error, implementation not allowed')
WHITE_SPACE ('\n')
WHITE_SPACE ('    ')
DlangTokenType.static ('static')
WHITE_SPACE (' ')
DlangTokenType.void ('void')
WHITE_SPACE (' ')
DlangTokenType.ID ('foo')
DlangTokenType.( ('(')
DlangTokenType.) (')')
WHITE_SPACE (' ')
DlangTokenType.{ ('{')
WHITE_SPACE (' ')
DlangTokenType.} ('}')
WHITE_SPACE (' ')
DlangTokenType.LINE_COMMENT ('// ok')
WHITE_SPACE ('\n')
WHITE_SPACE ('    ')
DlangTokenType.final ('final')
WHITE_SPACE (' ')
DlangTokenType.void ('void')
WHITE_SPACE (' ')
DlangTokenType.ID ('abc')
DlangTokenType.( ('(')
DlangTokenType.) (')')
WHITE_SPACE (' ')
DlangTokenType.{ ('{')
WHITE_SPACE (' ')
DlangTokenType.} ('}')
WHITE_SPACE (' ')
DlangTokenType.LINE_COMMENT ('// ok')
WHITE_SPACE ('\n')
DlangTokenType.} ('}')